### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by MrF3lix**
- [x] Frontend using Next.js \w Typescript
- [x] API using ASP.net Core
- [x] Executor using Node.js \w Typescript

### CODE REVIEW

**Comment by MrF3lix** [comment]
Once the setup is merged we can start setting up the GitHub workflows.

**Review by S4nvers — APPROVED** [review]
I don't really know react, but to me everything looks good

File: src/executor/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "typescript": "^4.6.2"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.13",
+    "@types/jest": "^27.4.1",
+    "@types/node": "^17.0.21",
+    "@typescript-eslint/eslint-plugin": "^5.14.0",
+    "@typescript-eslint/parser": "^5.14.0",
+    "eslint": "^8.10.0",
+    "eslint-config-airbnb-base": "^15.0.0",
```
Comment (riesentoaster) [inline src/executor/package.json:23]: Dude.

**Review by riesentoaster — CHANGES_REQUESTED** [review]

**Review by riesentoaster — APPROVED** [review]

**Review by DuplosFidibuss — APPROVED** [review]

**Review by silvan-flum — APPROVED** [review]
Tiptop, thanks.

**Review by nikipuk — APPROVED** [review]
Working for me now 🏎️

**Pull request merged**